### Hello, there! üëã

I‚Äôm Ankit, a passionate researcher with a diverse background in computational biology, quantitative modeling, and data analysis. Welcome to my GitHub profile!

üß¨ Current Work:

I'm currently immersed in the exciting world of machine learning, where I'm developing an innovative pipeline that seamlessly integrates single-cell RNA sequencing data with image-based spatial transcriptomics data. Through this integration of data sources, we're unraveling the intricate web of cellular crosstalk within various tissue niches. Our focus extends to deciphering the covariation of genes among interacting cell types. We've applied this method to publicly available datasets from liver, organogenesis, brain, and intestine tissues, with the aim of identifying novel therapeutic targets. (Stay tuned, as manuscripts and packages are in the works!)

ü¶¥ Past Endeavors:

In a previous project, I delved into the fascinating realm of bone growth morphogenesis. Using quantitative modeling, statistical analyses, and advanced morphometric techniques applied to growth plate tissue images, we unearthed novel insights into the mechanisms governing long bone elongation. Our journey included tasks such as cell/nuclei segmentation, bone registration, and the calculation of various morphological properties. Among our discoveries, we observed isometric growth of chondrocytes in the resting zone of the growth plate, contrasting with allometric growth in the proliferative zone. We also identified distinct patterns in cell lineage clusters between embryonic and neonatal mice.

To dive deeper into these discoveries, check out our publication: Application of 3D MAPs pipeline identifies the morphological sequence chondrocytes undergo and the regulatory role of GDF5 in this process. https://www.nature.com/articles/s41467-021-25714-0

üî¨ Previous Research Highlights:

I've explored various facets of biology and data analysis throughout my career. This includes using gene expression as a proxy for effective temperature to elucidate chromosome positioning in nuclei. I developed biophysical models to predict chromosome distributions in nuclei, even delving into their 2D/3D chromosomes shapes. My journey has also taken me into the realm of ChIP-seq data analysis to uncover mixtures of motifs using sequence similarity and position weight matrix models.

Interested in learning more? Dive into the details of my previous research through these publications:

Chromatin as active matter.
https://iopscience.iop.org/article/10.1088/1742-5468/aa5287

Nonequilibrium Biophysical Processes Influence the Large-Scale Architecture of the Cell Nucleus.
https://www.cell.com/biophysj/fulltext/S0006-3495(19)30940-3

THiCweed: fast, sensitive detection of sequence features by clustering big datasets.
https://academic.oup.com/nar/article/46/5/e29/4754463


üõ†Ô∏è Skills:

Throughout my research journey, I've acquired a versatile skill set spanning machine learning, quantitative biology, image analysis (including registration and segmentation), spatial biology, neighborhood analysis, and the quantification of cell/nuclei and tissue morphological properties (think volume, surface area, sphericity, principal component orientation, and many more).


üìÑ Learn More:

If you'd like to explore my credentials in more detail, feel free to peruse my CV.
https://github.com/ankitbioinfo/interestingLinks/blob/master/CV_ankit2.pdf

Thank you for visiting my GitHub profile! If you have any questions or want to collaborate on fascinating research projects, don't hesitate to reach out. Let's unlock the mysteries of biology and data together! üåü



a passionat currently working on a machine learning-based pipeline to integrate single-cell RNA sequencing data and image-based spatial transcriptomics data. With this data integration, we learned how the cell type-cell type interaction formed in the niche and how gene covariation occurs between interacted cell types. We have applied this method to the publically available liver, organogenesis, brain, and intestine data. This method could be beneficial for the community to find a novel therapeutic drug target candidate. (MS and packages under preparation)


Before this, I was involved in a project explaining how the long bone (tibia, ulna) elongates. We have described some novel facts about the bone growth mechanism with quantitative modeling, statistical analysis, and various morphometric analysis of cells and nuclei in growth plate (GP) tissue images. We have segmented the cells/nuclei, registered the different bones, and calculated the morphological properties such as volume, surface area, sphericity, PC ratio of cell axis, Radius of gyration, volume fraction, orientation analysis, and cre-labeled chondrocyte lineage clusters properties. In one of the observations, we found that in the resting zone of GP the chondrocyte cell grows isometric, while in the proliferative zone, they grow allometric. In other observations, we found that the cell lineage cluster grows circumferential in embryonic mice, while neonatal mice grow in the longitudinal direction. To read some of the  detail, please follow the link: 

(1) Application of 3D MAPs pipeline identifies the morphological sequence chondrocytes undergo and the regulatory role of GDF5 in this process 


Before the above, I used gene expression as a proxy for the effective temperature to explain the chromosome positioning in the nuclei. I developed a biophysical polymer model to predict the chromosome distributions in the nuclei and their 2d/3d shapes. The details of this study you can read it here: 

(2) Chromatin as active matter 

(3) Nonequilibrium Biophysical Processes Influence the Large-Scale Architecture of the Cell Nucleus

I also used ChIP-seq data to find the mixtures of motifs using sequence similarity and position weight matrix models. Please follow the link to read in detail: 

(4) THiCweed: fast, sensitive detection of sequence features by clustering big datasets

During my research career, I have learned many skills from different domains, such as machine learning concepts, quantitative biology, image registration, image segmentation, spatial biology, neighborhood analysis, and cell/nuclei and tissue morphological properties volume, surface area, sphericity, principal component ratio, orientation ratio, etc. 

Please find my CV if you are interested in more. 




<!--
**ankitbioinfo/ankitbioinfo** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.



Here are some ideas to get you started:

- üî≠ 



- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
