### Hello there! üëã

I‚Äôm Ankit, a researcher with a diverse background in computational biology, quantitative modeling, machine learning, and data analysis. Welcome to my GitHub profile!

üëâ üß™ Current Role
At CatalYm, I focus on the intersection of spatial omics, and immuno-oncology to uncover the cellular and molecular mechanisms that drive therapeutic resistance and immune evasion in the TME. My work integrates running different spatial-omics tools to accelerate discovery and guide translational decisions.


üëâ üß¨ Previous work (2021-2025) @ University of W√ºrzburg, Germany 
I developed NiCo, a computational pipeline that integrates scRNA-seq with image-based spatial transcriptomics to reveal cell-cell communication and gene program (meta-program) covariation within tissue niches. We applied NiCo to diverse biological contexts including liver, brain, and developmental tissues to uncover novel mechanistic insights.  

Check out our publication: 
- <ins>Decoding cell‚Äìcell communication using spatial transcriptomics. </ins>[Tool Highlight at Nature Reviews Genetics](https://www.nature.com/articles/s41576-025-00824-3)
- <ins>NiCo Identifies Extrinsic Drivers of Cell State Modulation by Niche Covariation Analysis.</ins> [Nature Communications 2024](https://www.nature.com/articles/s41467-024-54973-w) 

üëâ ü¶¥ Past Endeavors (2018-2020) @ Weizmann Institute of Science, Israel

I explored the fascinating realm of bone growth morphogenesis. Using quantitative modeling, statistical analyses, and morphometric techniques applied to growth plate tissue images, we uncovered mechanism governing long bone elongation. Some highlights are: Cell/Nuclei segmentation and registration of bones, Identification of isometric growth of chondrocytes in the resting zone vs. allometric growth in the proliferative zone. Discovery of distinct cell lineage cluster patterns between embryonic and neonatal mice bones contributing to growth directionality

To dive deeper into these discoveries, check out our publication: 

- <ins>Application of 3D MAPs pipeline identifies the morphological sequence chondrocytes undergo and the regulatory role of GDF5 in this process.</ins> [Nature Communications 2021](https://www.nature.com/articles/s41467-021-25714-0)
- <ins>Limited column formation in the embryonic growth plate implies divergent growth mechanisms during pre- and postnatal bone development.</ins> [eLife 2024](https://elifesciences.org/articles/95289)


üëâ üî¨ Research Highlights (2013-2018) @ The Institute of Mathematical Sciences, Chennai, India 

Earlier in my research career, I worked at the intersection of molecular biology, genomics, and computational modeling. Key projects included:

- Chromosome Positioning & Nuclear Architecture
  Developed biophysical models to predict chromatin distribution and chromosome positioning along the radial nuclear axis using gene expression data as a proxy. Explored 2D/3D chromosome structures to understand spatial organization within nuclei.

- Gene Regulation & ChIP-seq Analysis
  Analyzed ChIP-seq datasets to identify motif mixtures using clustering based on sequence similarity and position-based features, helping to characterize regulatory element patterns.

These foundational projects strengthened my skills in statistical modeling, data integration, and the use of genomic signals to inform structural and regulatory insights in cellular systems.


Interested in learning more? Dive into the details of my previous research through these publications:

- <ins>Chromatin as active matter.</ins> [JSTAT 2017](https://iopscience.iop.org/article/10.1088/1742-5468/aa5287)
- <ins>Nonequilibrium Biophysical Processes Influence the Large-Scale Architecture of the Cell Nucleus.</ins> [Biophysical Journal 2020](https://www.cell.com/biophysj/fulltext/S0006-3495(19)30940-3)
- <ins>THiCweed: fast, sensitive detection of sequence features by clustering big datasets.</ins> [Nucleic Acid Research 2017](https://academic.oup.com/nar/article/46/5/e29/4754463)


üëâ üõ†Ô∏è Skills:

Throughout my research journey, I've acquired a versatile skill set spanning machine learning, molecular dynamics simulation, quantitative and statistical data modeling, image analysis, spatial biology, spatial transcriptomics, scRNA-seq, cellular neighborhood analysis, and the quantification of cell/nuclei morphological properties (volume, surface area, sphericity, principal component orientation, and many more).


üëâ üìÑ Learn More:

If you'd like to explore my credentials in more detail, feel free to contact me on [linkedin](https://www.linkedin.com/in/ankit-agrawal-spatial-omics/)

Thank you for visiting my GitHub profile! If you have any questions or want to collaborate on fascinating research projects, don't hesitate to reach out. Let's unlock the mysteries of biology and data together! üåü

[ORCID: 0009-0006-1700-2397](https://orcid.org/0009-0006-1700-2397)

W 1234#
Scopus Author ID: 57193255954




<!--
**ankitbioinfo/ankitbioinfo** is a ‚ú® _special_ ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.



Here are some ideas to get you started:

- üî≠ 



- üå± I‚Äôm currently learning ...
- üëØ I‚Äôm looking to collaborate on ...
- ü§î I‚Äôm looking for help with ...
- üí¨ Ask me about ...
- üì´ How to reach me: ...
- üòÑ Pronouns: ...
- ‚ö° Fun fact: ...
-->
